src/test/java/org/example/UserServiceTest.java

```diff
@@ -0,0 +1,121 @@
+package org.example;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class UserServiceTest {
+
+    private final UserService userService = new UserService();
+
+    private final UserGenerator userGenerator = new UserGenerator();
+
+    private final List<Client> clientList = userGenerator.getClients();
+
+    @Test
+    void should_getSumByAge() {
+        //given
+        String name = "bob";
+
+        //when
+        int sum = userService.getSumAgeByName(clientList, name);
+
+        //then
+        int expectedSum = 820;
+        assertEquals(expectedSum, sum);
+    }
+
+    @Test
+    void should_getNamesByMention() {
+        //when
+        Set<String> set = userService.getNamesByMention(clientList);
+
+        //then
+        Set<String> expectedSet = new LinkedHashSet<>(Set.of("bob", "kek", "a", "b", "c", "tom", "d", "bruh", "greg"));
+        assertEquals(set, expectedSet);
+    }
+
+    @Test
+    void should_returnTrueIfAgeGreaterExists() {
+        //when
+        boolean isExist = userService.doesAgeGreaterExist(clientList,0);
```

Comment:
instead of passing only 1 parameter, send several to be sure it works for different cases:
https://www.baeldung.com/parameterized-tests-junit-5